x-pack/plugins/security_solution/public/explore/hosts/components/uncommon_process_table/index.tsx

```diff
@@ -141,85 +130,6 @@ export const UncommonProcessTable = React.memo(UncommonProcessTableComponent);
 
 UncommonProcessTable.displayName = 'UncommonProcessTable';
 
-const getUncommonColumns = (): UncommonProcessTableColumns => [
-  {
-    name: i18n.NAME,
-    truncateText: false,
-    mobileOptions: { show: true },
-    width: '20%',
-    render: ({ node }) =>
-      getRowItemsWithActions({
-        values: node.process.name,
-        fieldName: 'process.name',
-        idPrefix: `uncommon-process-table-${node._id}-processName`,
-      }),
-  },
-  {
-    align: 'right',
-    name: i18n.NUMBER_OF_HOSTS,
-    truncateText: false,
-    mobileOptions: { show: true },
-    render: ({ node }) => <>{node.hosts != null ? node.hosts.length : getEmptyValue()}</>,
-    width: '8%',
-  },
-  {
-    align: 'right',
-    name: i18n.NUMBER_OF_INSTANCES,
-    truncateText: false,
-    mobileOptions: { show: true },
-    render: ({ node }) => defaultToEmptyTag(node.instances),
-    width: '8%',
-  },
-  {
-    name: i18n.HOSTS,
-    truncateText: false,
-    mobileOptions: { show: true },
-    width: '25%',
-    render: ({ node }) =>
-      getRowItemsWithActions({
-        values: getHostNames(node.hosts),
-        fieldName: 'host.name',
-        idPrefix: `uncommon-process-table-${node._id}-processHost`,
-        render: (item) => <HostDetailsLink hostName={item} />,
-      }),
-  },
-  {
-    name: i18n.LAST_COMMAND,
-    truncateText: false,
-    mobileOptions: { show: true },
-    width: '25%',
-    render: ({ node }) =>
-      getRowItemsWithActions({
-        values: node.process != null ? node.process.args : null,
-        fieldName: 'process.args',
-        idPrefix: `uncommon-process-table-${node._id}-processArgs`,
-        render: (item) => <HostDetailsLink hostName={item} />,
-        displayCount: 1,
-      }),
-  },
-  {
-    name: i18n.LAST_USER,
-    truncateText: false,
-    mobileOptions: { show: true },
-    render: ({ node }) =>
-      getRowItemsWithActions({
-        values: node.user != null ? node.user.name : null,
-        fieldName: 'user.name',
-        idPrefix: `uncommon-process-table-${node._id}-processUser`,
-      }),
-  },
-];
-
-export const getHostNames = (hosts: HostEcs[]): string[] => {
-  if (hosts != null) {
-    return hosts
-      .filter((host) => host.name != null && host.name[0] != null)
-      .map((host) => (host.name != null && host.name[0] != null ? host.name[0] : ''));
-  } else {
-    return [];
-  }
-};
-
 export const getUncommonColumnsCurated = (pageType: HostsType): UncommonProcessTableColumns => {
```

Comment:
Good idea extracting the columns logic to _uncommon_process_table/columns.tsx_ file, I wonder if it would make sense to move the `getUncommonColumnsCurated` there as well. By doing so we would encapsulate everything "column-related" in the columns.tsx file, we could also move the tests to new _columns.test.tsx_.